Dictionary.json

```diff
@@ -2338,7 +2338,76 @@
 		"Tips5" : "https://github.com/ibhavikmakwana/FlutterDartTips",
 		"Tips6" : "https://github.com/leisim/awesome-flutter-packages",
 		"Tips7" : "https://medium.com/@diegoveloper"
-	},	
+	},
+{
+    "Title": "Frontend Developer",
+    "Tag": "Web development",
```

Comment:
change to "Development"  only as frontend is also there is other applications also